pkg/services/provisioning/provisioning.go

```diff
@@ -113,6 +124,12 @@ func (ps *provisioningServiceImpl) ProvisionNotifications() error {
 	return errutil.Wrap("Alert notification provisioning error", err)
 }
 
+func (ps *provisioningServiceImpl) ProvisionUsers() error {
+	usersPath := path.Join(ps.Cfg.ProvisioningPath, "users")
+	err := ps.provisionUsers(usersPath)
+	return errutil.Wrap("Alert notification provisioning error", err)
```

Comment:
This error needs to change to something: `User provisioning error`

Reply:
Moving my suggestion here so we don't discuss the same thing in two different places 🙂 

```suggestion
	return errutil.Wrap("Users provisioning error", err)
```